### PR TITLE
Fixes the Dockerfiles for Seth development builds

### PR DESCRIPTION
**Pull request by grkvlt**
Updates to the documentation at the start of the Seth Dockerfiles, showing build and run commands.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -63,9 +63,9 @@
 master_doc = 'contents'
 
 # General information about the project.
-project = u'Sawtooth'
-copyright = u'2015-2017, Intel Corporation'
-author = u'Intel Corporation'
+project = u'Sawtooth Seth'
+copyright = u'2017-2018'
+author = u'Hyperledger Foundation'
```
Comment (dcmiddle) [inline docs/source/conf.py]: Awaiting clarification from Linux Foundation / Hyperledger on the copyright policy.

Comment (grkvlt) [reply docs/source/conf.py]: I'll revert these for now, so we can get this merged sooner @dcmiddle

Comment (dcmiddle) [reply docs/source/conf.py]: sounds good

File: docs/source/introduction.rst [hunk 2]
```diff
@@ -44,47 +44,28 @@ in a *transaction family*. While we expect users to build custom transaction
 families that reflect the unique requirements of their ledgers, we provide
 several core transaction families as models\:
 
-    * Validator Registry - Provides a methodology
+    * *Validator Registry* - Provides a methodology
```
Comment (aludvik) [inline docs/source/introduction.rst]: This introduction needs to be rewritten and all of these Sawtooth-specific references deleted

Comment (grkvlt) [reply docs/source/introduction.rst]: Agree, but that can be done in a separate pull request, with a complete documentation update/rewrite

File: docs/source/conf.py (hunk 1 above)
Comment (grkvlt) [reply docs/source/conf.py]: @dcmiddle I have reverted back to using the original copyright notices

**Comment by grkvlt** [comment]
On Ubuntu 16.04.4 LTS with Docker 17.12.1-ce the image fails to build with the following error:
```
 ---> 37b083f402f7
Step 7/13 : RUN mkdir /go  && go get -u     github.com/golang/protobuf/proto     github.com/golang/protobuf/protoc-gen-go     github.com/pebbe/zmq4     github.com/brianolson/cbor_go     github.com/satori/go.uuid     github.com/btcsuite/btcd/btcec     github.com/btcsuite/btcutil/base58     gopkg.in/fatih/set.v0     golang.org/x/crypto/ripemd160     github.com/jessevdk/go-flags     github.com/pelletier/go-toml     golang.org/x/crypto/ssh/terminal
 ---> Running in 11919f5d4c1c
package math/bits: unrecognized import path "math/bits" (import path does not begin with hostname)
The command '/bin/sh -c mkdir /go  && go get -u     github.com/golang/protobuf/proto     github.com/golang/protobuf/protoc-gen-go     github.com/pebbe/zmq4     github.com/brianolson/cbor_go     github.com/satori/go.uuid     github.com/btcsuite/btcd/btcec     github.com/btcsuite/btcutil/base58     gopkg.in/fatih/set.v0     golang.org/x/crypto/ripemd160     github.com/jessevdk/go-flags     github.com/pelletier/go-toml     golang.org/x/crypto/ssh/terminal' returned a non-zero code: 1

```
/cc @dplumb94

**Comment by grkvlt** [comment]
@dplumb94 I think I can close this now, right?

**Comment by dcmiddle** [comment]
Are you looking to merge or close @grkvlt.  Sorry I had missed that you had reverted the copyright stuff. I've just updated my approval flag for the PR based on that.

**Comment by dplumb94** [comment]
The build process has been changed so those Dockerfile changes shouldn't be required anymore. Do we still want to save the documentation changes?

**Comment by grkvlt** [comment]
Closing since superseded by #14 but will incorporate documentation changes into another PR shortly

**Pull request closed without merging**